promql/query_logger_test.go

```diff
@@ -77,16 +78,16 @@ func TestIndexReuse(t *testing.T) {
 	}
 
 	queryLogger.generateIndices(3)
-	queryLogger.Insert("TestQuery1")
-	queryLogger.Insert("TestQuery2")
-	queryLogger.Insert("TestQuery3")
+	queryLogger.Insert(context.TODO(), "TestQuery1")
```

Comment:
You can use background for tests. It's not as if we'll be changing it in future.

Reply:
I will fix it once I have added proper tests